tests/components/mysensors/__init__.py

```diff
@@ -0,0 +1,212 @@
+"""Test MySensors component."""
+from enum import IntEnum
+from unittest.mock import MagicMock
+
+import voluptuous as vol
+
+from homeassistant.components import mysensors as mysensors_comp
+from homeassistant.setup import async_setup_component
+from tests.common import mock_coro_func
+
+DEVICE = '/dev/ttyACM0'
+
+
+def get_gateway(hass):
+    """Helper to get a gateway from set up gateways."""
+    gateways = hass.data[mysensors_comp.MYSENSORS_GATEWAYS]
+    gateway = next(iter(gateways.values()))
+    return gateway
+
+
+async def setup_mysensors(hass):
+    """Set up mysensors."""
+    config = {'mysensors': {
+        'gateways': [{'device': DEVICE}],
+        'version': '2.0', 'persistence': False}}
+
+    res = await async_setup_component(hass, 'mysensors', config)
+    return res
+
+
+class MessageType(IntEnum):
+    """MySensors message types."""
+
+    # pylint: disable=too-few-public-methods
+    presentation = 0
+    set = 1
+    req = 2
+    internal = 3
+    stream = 4
+
+
+class Presentation(IntEnum):
+    """MySensors presentation sub-types."""
+
+    # pylint: disable=too-few-public-methods
+    S_BINARY = 3
+    S_DIMMER = 4
+    S_TEMP = 6
+    S_LIGHT_LEVEL = 16
+    S_SOUND = 33
+    S_VIBRATION = 34
+
+
+class SetReq(IntEnum):
```

Comment:
Copying these values from the package seems weird. It's prone to get out of sync. It would be better to import it directly from the package.